src/engine/renderer/tr_local.h

```diff
@@ -1736,97 +1734,53 @@ enum class ssaoMode {
 	};
 
 // ydnar: plain map drawsurfaces must match this header
-	struct srfGeneric_t
-	{
+	struct srfGeneric_t {
 		surfaceType_t surfaceType;
 
-		// culling information
-		vec3_t   bounds[ 2 ];
-		vec3_t   origin;
-		float    radius;
-	};
+		// Culling information
+		vec3_t bounds[2];
+		vec3_t origin;
+		float radius;
 
-	struct srfGridMesh_t : srfGeneric_t
-	{
-		// lod information, which may be different
-		// than the culling information to allow for
-		// groups of curves that LOD as a unit
-		vec3_t lodOrigin;
-		float  lodRadius;
-		int    lodFixed;
-		int    lodStitched;
+		cplane_t plane;
 
-		// triangle definitions
-		int           width, height;
-		float         *widthLodError;
-		float         *heightLodError;
+		int numVerts;
```

Comment:
Please add comments to indicate which surface types each field is defined in. It's really annoying to have fields that are only valid in a random subset of the objects and not know which ones.